CoffeeChat/Controller/EditDemographicsViewController.swift

```diff
@@ -0,0 +1,284 @@
+//
+//  EditDemographicsViewController.swift
+//  CoffeeChat
+//
+//  Created by Lucy Xu on 4/12/20.
+//  Copyright © 2020 cuappdev. All rights reserved.
+//
+
+import UIKit
+
+class EditDemographicsViewController: UIViewController {
+
+    // MARK: - Private Data Vars
+    // TODO: Get values from backend? Or store in UserDefaults?
+    private let name = "Lucy Xu"
+    private let year = "2021"
+    private let major = "Computer Science"
+    private let hometown = "Boston, MA"
+    private let pronouns = "She/Her/Hers"
```

Comment:
we should just refetch each time. im assuming backend is giving you a token to make requests upon login. once they login, you should save the token in `userDefaults` so that they don't have to relogin